eng/references.targets

```diff
@@ -1,67 +1,58 @@
 <Project>
   <PropertyGroup>
-    <ContractDependencyPaths>$(RefPath)</ContractDependencyPaths>
-    <FrameworkPathOverride>$(RefPath)</FrameworkPathOverride>
-    <AssemblySearchPaths>$(AssemblySearchPaths);$(RefPath);{RawFileName}</AssemblySearchPaths>
+    <IncludeDefaultReferences Condition="'$(IncludeDefaultReferences)' == ''">true</IncludeDefaultReferences>
     <!-- Disable RAR from transitively discovering dependencies for References -->
     <_FindDependencies>false</_FindDependencies>
-    <!--
-      We don't use any of MSBuild's resolution logic for resolving the framework, so just set these two properties to any folder
-      that exists to skip the GenerateReferenceAssemblyPaths task (not target) and to prevent it from outputting a warning (MSB3644).
-      Need to set these after the common targets import.
-      -->
-    <_TargetFrameworkDirectories>$(MSBuildThisFileDirectory)</_TargetFrameworkDirectories>
-    <_FullFrameworkReferenceAssemblyPaths>$(MSBuildThisFileDirectory)</_FullFrameworkReferenceAssemblyPaths>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(IncludeDefaultReferences)' == ''">
-    <IncludeDefaultReferences Condition="'$(MSBuildProjectExtension)' == '.csproj'">true</IncludeDefaultReferences>
-    <IncludeDefaultReferences Condition="'$(MSBuildProjectExtension)' == '.vbproj'">true</IncludeDefaultReferences>
   </PropertyGroup>
 
-  <Target Name="SetupDefaultReferences">
-    <ItemGroup Condition="'$(IncludeDefaultReferences)' == 'true'">
-      <!-- netstandard is a default reference whenever building for NETStandard or building an implementation assembly -->
-      <DefaultReference Condition="('$(TargetFrameworkIdentifier)' == '.NETStandard' or '$(IsReferenceAssembly)' != 'true') and Exists('$(RefPath)netstandard.dll')" Include="netstandard" />
-    </ItemGroup>
-  </Target>
-  <Target Name="UpdateReferenceItems" DependsOnTargets="SetupDefaultReferences" BeforeTargets="AddReferencesDynamically">
-    <ItemGroup>
-      <Reference Include="@(DefaultReference)" />
-    </ItemGroup>
-    <ItemGroup>
-      <!-- Simple name references will be resolved from the targeting pack folders and should never be copied to output -->
-      <Reference Condition="'%(Reference.Extension)' != '.dll'">
-        <Private>false</Private>
-      </Reference>
-    </ItemGroup>
-  </Target>
+  <!-- netstandard is a default reference whenever building an implementation assembly for .NETCoreApp. -->
```

Comment:
Why is this? This was not the behavior we had, was it?

Reply:
Before we added the reference to both .NETFramework and .NETCoreApp source projects but with @joperezr recent work this isn't necessary anymore and actually shouldn't be done anymore so that the netfx libraries work without the .NETStandard shims.